scripts/workflow_config/sync_testnets_matrix.json

```diff
@@ -35,8 +35,40 @@
     "network": "mainnet",
     "cl": "lodestar",
     "cl_image": "chainsafe/lodestar:latest",
-    "checkpoint-sync-url": "https://mainnet.checkpoint.sigp.io",
+    "checkpoint-sync-url": "https://mainnet-checkpoint-sync.attestant.io/",
     "timeout": 180,
     "agent": "g6-standard-16"
+  },
+  {
+    "network": "op-sepolia",
+    "cl": "",
+    "cl_image": "",
+    "checkpoint-sync-url": "",
+    "timeout": 180,
+    "agent": "g6-standard-8"
+  },
+  {
+    "network": "base-sepolia",
+    "cl": "",
+    "cl_image": "",
+    "checkpoint-sync-url": "",
+    "timeout": 180,
+    "agent": "g6-standard-8"
+  },
+  {
+    "network": "op-mainnet",
+    "cl": "",
+    "cl_image": "",
+    "checkpoint-sync-url": "",
+    "timeout": 600,
+    "agent": "g6-standard-16"
```

Comment:
I wonder if we really need such a big VM here.

Reply:
As still on draft - testing what machines shall I use - will add debug "du -h" at the end to see exactly what types of machines shall I use.

Also will test out if I can use NonValidatorNode for op stack - if works will be definitelly smaller